application/models/post.go

```diff
@@ -363,6 +376,13 @@ func (p *Post) ValidateCreate(tx *pop.Connection) (*validate.Errors, error) {
 	//return validate.NewErrors(), nil
 }
 
+type futureDateValidator struct {
```

Comment:
Is this still needed?

Reply:
Oh yeah. I found the built in validator after I started on that.